_includes/guide/what_is_api.html

```diff
@@ -0,0 +1,9 @@
+<p>
+  The Beneficiary Claims Data API (BCDA) enables Accountable Care Organizations (ACOs) to retrieve claims data for their Medicare beneficiaries. This includes claims data for instances in which beneficiaries receive care outside of the ACO, allowing a full picture of patient care.
+</p>
+<p>
+  This API follows the workflow outlined by the <a href="https://github.com/HL7/bulk-data/blob/master/spec/export/index.md" target="_blank">FHIR Bulk Data Export Proposal</a> using the HL7 FHIR Standard. Claims data is provided as FHIR resources in <a href="http://ndjson.org/" target="_blank">NDJSON</a> format.
```

Comment:
Should we link out to the hl7 page instead of the github page? https://hl7.org/fhir/uv/bulkdata/index.html

Reply:
The link to the github page was in the approved language for the quickstart page